src/components/TopBar/CallTime.vue

```diff
@@ -185,6 +195,18 @@ export default {
 		clearInterval(this.timer)
 	},
 
+	watch: {
+
+		callTime(value) {
+			if (value && !this.untilCallDurationHintShown) {
+				this.untilCallDurationHintShown = (1000 * 35) - value + 1
```

Comment:
```suggestion
				const untilCallDurationHintShown = (1000 * 60 * 60) - value + 1
```
`const` is not working here?